datafusion/sqllogictest/test_files/string/string_query.slt.part

```diff
@@ -1096,3 +1096,20 @@ Rapha Raphael NULL datafusionДатаФус datafusionДатаФусион NULL
 und under_score NULL un iść core un iść core NULL
 p percent NULL pan Tadeusz ma iść w kąt pan Tadeusz ma iść w kąt NULL
 NULL NULL NULL NULL NULL NULL
+
+# TODO: Support Utf8View for bit_length array string function
```

Comment:
I don't understand this comment -- it seems like bit_length is working on Utf8View in datafusion/sqllogictest/test_files/string/string_view.slt

If you just uncomment this code out that would mean we are covering bit_length for all the string types

Reply:
Thanks @alamb 
As I understand it, applying `bit_length()` to a column will currently fail because `arrow::compute::kernels::length::bit_length` does not yet support UTF-8 views (depends on [PR](https://github.com/apache/arrow-rs/pull/6671) in `arrow-rs`). However, it's still used in the following code path:
https://github.com/apache/datafusion/blob/a16854837a826ad60d9e254b75b6135a076f6545/datafusion/functions/src/string/bit_length.rs#L74

Reply:
To cover other string types, we can add this test to each string testing file before https://github.com/apache/arrow-rs/pull/6671.